packages/blade/src/components/Accordion/_decisions/decisions.md

```diff
@@ -50,16 +50,16 @@ import { Accordion, AccordionItem } from '@razorpay/blade';
 | -------------------- | ---------- | ----------- | -------------------------------------------------------------- | -------- |
 | defaultExpandedIndex | `number`   | `undefined` | Makes the passed item index expanded by default (uncontrolled) |          |
 | expandedIndex        | `number`   | `undefined` | Expands the passed index (controlled)                          |          |
-| onChange             | `function` | `undefined` | Callback for change in any item's expanded state               |          |
+| onExpandChange       | `function` | `undefined` | Callback for change in any item's expanded state               |          |
```

Comment:
Should we call this onOpenChange as well? refer to discussion above for the same on Collapsible

Reply:
Related discussion [here](https://github.com/razorpay/blade/pull/1218#discussion_r1226101854). Overall, this is to make it more contextual to collapsible kind of interactions.